addons/website/views/snippets.xml

```diff
@@ -1014,13 +1008,23 @@
 <template id="snippet_options">
     <t t-call="web_editor.snippet_options"/>
 
-    <!-- COLOR | .s_three_columns | .s_comparisons -->
+    <!-- COLOR, BORDER, SHADOW | .s_three_columns | .s_comparisons -->
     <div data-selector=".s_three_columns .row > div, .s_comparisons .row > div, .s_tabs .row > div"
          data-target=".card">
-        <we-colorpicker string="Color"
+        <we-colorpicker string="Background Color"
             data-select-style="true"
             data-css-property="background-color"
             data-color-prefix="bg-"/>
+        <we-row string="Border" class="o_we_inline">
+            <we-colorpicker data-select-style="true" data-css-property="border-color" data-color-prefix="border-"/>
+            <we-input class="o_we_inline"
+                      data-select-style="0" data-css-property="border-width"
+                      data-unit="px" data-extra-class="border"/>
+        </we-row>
+        <we-input string="Rounded" class="o_we_inline"
+                  data-select-style="0" data-css-property="border-radius"
+                  data-unit="px" data-extra-class="rounded"/>
+        <we-checkbox string="Shadow" data-select-class="shadow"/>
```

Comment:
Mmh, I guess we will have to share some code in a sub-template or something... Justa thought, maybe we can do that later

Reply:
Considering the time it takes, I might as well just do it. I'll revert it if sbu says we don't need these on the three columns cards.